Algorithm.CSharp/CorrectConsolidatedBarTypeForTickTypesAlgorithm.cs

```diff
@@ -29,7 +29,6 @@ public class CorrectConsolidatedBarTypeForTickTypesAlgorithm : QCAlgorithm, IReg
     {
         private bool _quoteTickConsolidatorCalled;
         private bool _tradeTickConsolidatorCalled;
-        private bool _openInterestTickConsolidatorCalled;
```

Comment:
Why was the open interest tick consolitator code removed here and in the Python algo?

Reply:
Oh because Open Interest is just for futures and options, not for equities